### PR TITLE
Issue266

### PR DESCRIPTION
**Pull request by khanzadyan**
Fixes https://github.com/elasticio/docs/issues/266

### CODE REVIEW

**Comment by khanzadyan** [comment]
FYI @zubairov

File: content/_getting-started/contracts-and-workspaces.md [hunk 1]
```diff
@@ -110,19 +117,19 @@ client’s engineers, and production environment is customer-facing only.
 
 ## Limited Workspaces
 
-Workspaces can have the `limited` or `full` type. `Limited` is intended for platform trial period. Workspaces with `limited` type have certain restrictions for integration flows:
+The limited workspaces have certain restrictions for integration flows:
 
-- Integration flows in `limited` workspaces are restricted by work time. This means that they are automatically stopped after a certain time. This time is defined for each installation with an environment variable.
+*   Integration flows in the limited workspaces are restricted by work time. Meaning, the **platform automatically stops all active flows after 8 hours** in limited workspaces. You can restart the flows afterwards to continue your testing.
+*   If your integration flow contains messages in the processing queues, these messages will be removed when the time is up to stop the flow.
```
Comment (jhorbulyk) [inline content/_getting-started/contracts-and-workspaces.md:123]: * Drop the word `Meaning`
 * Consider adding the word `However,` to the start of the second bullet point and then pull it into the first paragraph.

File: content/_getting-started/contracts-and-workspaces.md [hunk 2]
```diff
@@ -110,19 +117,19 @@ client’s engineers, and production environment is customer-facing only.
...
 
-- In `limited` workspaces, integration flows that contain errors will not be suspended. Instead, they will be stopped without saving any data.
+> We recommend not to use **limited workspaces** for the production purposes. Use **full workspaces** instead.
 
-Though not forbidden, **it is highly unrecommended to use `limited` workspaces for any production purposes**. `Limited` workspaces are indicated in the UI:
+The limited workspaces are clearly marked in the platform UI:
```
Comment (jhorbulyk) [inline content/_getting-started/contracts-and-workspaces.md:127]: * I would avoid the use of superfluous adjectives such as *clearly*, *obviously*, etc.

File: content/_getting-started/contracts-and-workspaces.md [hunk 3]
```diff
@@ -110,19 +117,19 @@ client’s engineers, and production environment is customer-facing only.
...
 
 {% include img.html max-width="100%" url="/assets/img/getting-started/contracts-and-workspaces/limited-workspace.png" title="Workspace - Limited" %}
 
-To change workspace type from `limited` to `full`, contact support.
-
-The default type for workspaces created in a tenant can be changed. A user with corresponding permission can do that, using `"default_workspace_type"`, setting `limited` or `full` as a value.
+To change workspace type from **limited** to **full**, contact our support or
```
Comment (jhorbulyk) [inline content/_getting-started/contracts-and-workspaces.md:131]: * contact our support **team**

**Review by jhorbulyk — COMMENTED** [review]
@khanzadyan Here is my subjective feedback on writing styles. Feel free to take/leave them as you see fit.

File: content/_getting-started/contracts-and-workspaces.md [hunk 4]
```diff
@@ -80,6 +80,13 @@ members with the corresponding permissions. Members can contribute to
 integration flows in their workspace in collaboration with other members, or
 individually.
 
+There are two types of workspaces: **limited** and **full**. We use full workspaces
+for the production purposes and limited workspaces for the platform trial periods
+or developer projects.
```
[the remark this answers is not in the document]
Comment (khanzadyan) [reply content/_getting-started/contracts-and-workspaces.md:85]: Agreed.

File: content/_getting-started/contracts-and-workspaces.md [hunk 5]
```diff
@@ -80,6 +80,13 @@ members with the corresponding permissions. Members can contribute to
...
+
+> The **full** workspaces have no time restrictions on integration flows. Check the
```
[the remark this answers is not in the document]
Comment (khanzadyan) [reply content/_getting-started/contracts-and-workspaces.md:87]: OK, good point.

File: content/_getting-started/contracts-and-workspaces.md [hunk 6]
```diff
@@ -110,19 +117,19 @@ client’s engineers, and production environment is customer-facing only.
 
 ## Limited Workspaces
 
-Workspaces can have the `limited` or `full` type. `Limited` is intended for platform trial period. Workspaces with `limited` type have certain restrictions for integration flows:
+The limited workspaces have certain restrictions for integration flows:
```
[the remark this answers is not in the document]
Comment (khanzadyan) [reply content/_getting-started/contracts-and-workspaces.md:120]: ok

File: content/_getting-started/contracts-and-workspaces.md (hunk 1 above)
Comment (khanzadyan) [reply content/_getting-started/contracts-and-workspaces.md:123]: Yes, to first. However, using a `However` in the beginning of the bullet point would not be suitable. IMHO.

File: content/_getting-started/contracts-and-workspaces.md (hunk 2 above)
Comment (khanzadyan) [reply content/_getting-started/contracts-and-workspaces.md:127]: ok

File: content/_getting-started/contracts-and-workspaces.md (hunk 3 above)
Comment (khanzadyan) [reply content/_getting-started/contracts-and-workspaces.md:131]: ok

File: content/_getting-started/contracts-and-workspaces.md (hunk 1 above)
Comment (jhorbulyk) [reply content/_getting-started/contracts-and-workspaces.md]: Perhaps then create 3 bullet points with *You can restart the flows afterwards to continue your testing.* being it's own point.

**Comment by khanzadyan** [comment]
@jhorbulyk have a look at it and if you agree please accept it.

File: content/_getting-started/contracts-and-workspaces.md (hunk 1 above)
Comment (khanzadyan) [reply content/_getting-started/contracts-and-workspaces.md]: aha, good point - one moment will implement you suggestion

Comment (khanzadyan) [reply content/_getting-started/contracts-and-workspaces.md]: Check the result please

**Review by jhorbulyk — APPROVED** [review]

**Pull request merged**